frontend/src/components/HomePage/index.jsx

```diff
@@ -4,12 +4,15 @@ import './HomePage.css';
 import Card from './Card';
 import Banner from './Banner';
 import Filters from './Filters';
+import { createQueryStr } from '../../utils';
 
 const HomePage = () => {
   const [bills, setBills] = useState([]);
+  const [searchTerm, setSearchTerm] = useState('');
```

Comment:
```suggestion
  const [searchTerm, setSearchTerm] = useState('*');
```

Since the default in `Filters.jsx` is `"*"`